pom.xml

```diff
@@ -1470,5 +1470,31 @@
       </build>
     </profile>
 
+    <profile>
+      <id>pitest-size</id>
```

Comment:
Actualy we can't added all packages to pites because there are some problems with some packages. For example "checks.annotation" failed build with ArrayIndexOutOfBoundsException. Your issue: https://github.com/hcoles/pitest/issues/302.
Testing "grammars" package takes a lot of time (> 15 min) and it write many errors in log but it finished succesfully. I suppose it connected with JDK 8 features.
I agree with idea of renaming profiles. That's good.